packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -31,16 +34,38 @@ export const sectionFooterLeftStyles = {
 
 const styles = (theme: ThemeType): JssStyles => ({
   profilePage: {
+    display: 'grid',
+    gridTemplateColumns: `1fr ${SECTION_WIDTH}px 1fr`,
+    gridTemplateAreas: `
+      '. center right'
+    `,
+    justifyContent: 'center',
+    columnGap: 40,
     marginLeft: "auto",
+    [theme.breakpoints.down('md')]: {
+      gridTemplateColumns: `${SECTION_WIDTH}px 1fr`,
+      gridTemplateAreas: `
+        'center right'
+      `,
+      paddingLeft: 20,
+      paddingRight: 20
+    },
     [theme.breakpoints.down('sm')]: {
+      display: 'block',
       margin: 0,
     }
   },
```

Comment:
Ok, so in order to build the righthand sidebar on this page without having to rebuild the whole layout, I decided to wrap the existing page in a grid. I think it works ok, but if you have a better suggestion let me know.

Also open to suggestions on different ways to lay out the new profile info.